libctru/source/services/apt.c

```diff
@@ -911,6 +911,7 @@ void aptLaunchLibraryApplet(NS_APPID appId, void* buf, size_t bufsize, Handle ha
 
 void aptLaunchSystemApplet(NS_APPID appId, void* buf, size_t bufsize, Handle handle)
 {
+	aptDspSleep();
```

Comment:
I believe the call to aptDspSleep should be placed between GSPGPU_SaveVramSysArea and GSPGPU_ReleaseRight, just like in aptJumpToHomeMenu.

Reply:
you are right, fixed (tested and pushed) this now